transaction.js

```diff
@@ -10,6 +10,7 @@ var MessageStream = require('./messagestream');
 var trans = exports;
 
 var MAX_HEADER_LINES = config.get('max_header_lines') || 1000;
+var REMOVE_CARRIAGE_RETURN = config.get('remove_carriage_return') || 'true';
```

Comment:
Also, you're setting REMOVE_CARRIAGE_RETURN='true' (a string that contains 'true'), which really really really ought to be the boolean `true` instead.